java/ql/src/Security/CWE/CWE-643/XPathInjection.qhelp

```diff
@@ -22,14 +22,14 @@ XPath injection can also be prevented by using XQuery.
 
 <example>
 <p>
-In the first, second, and third example, the code accepts a name and password specified by the user, and uses this
+In the first three examples, the code accepts a name and password specified by the user, and uses this
 unvalidated and unsanitized value in an XPath expression. This is vulnerable to the user providing
 special characters or string sequences that change the meaning of the XPath expression to search
 for different values.
 </p>
 
 <p>
-In the fourth example, the code utilizes setXPathVariableResolver which prevents XPath Injection.
+In the fourth example, the code uses `setXPathVariableResolver` which prevents XPath injection.
```

Comment:
Sorry - this is an error from my PR 🤦🏻 

Too much time spent writing markdown recently!

```suggestion
In the fourth example, the code uses <code>setXPathVariableResolver</code> which prevents XPath injection.
```

Reply:
No worries, I should've caught it before rebasing.